### DISCUSSION TITLE
Activate 5G in ORC8R Docker

### DISCUSSION THREAD

**Discussion by alaabouz** [magma/magma#14846] [Q&A] [answered]
Hello All,

Could you advise where can i find a doc on how to activate the 5G SA in the orc8r, i found this discussion
https://github.com/magma/magma/issues/9962
but i am not sure in which dockder container i have to do this modifications.

Thanks in advance for your support !

Br,
Alaa.

**Comment by panyogesh** [comment, marked as answer]
We can use swagger or NMS UI to do that. Please find a quick reference for the same [enable-5g](https://magma.github.io/magma/docs/lte/integrated_5g_sa)

**Comment by panyogesh** [comment]
Closing as its quite old.
